lib/msal-angular/package.json

```diff
@@ -24,7 +24,7 @@
     "node": ">=0.8.0"
   },
   "dependencies": {
-    "msal": "0.2.1",
+    "msal": "0.2.2",
```

Comment:
why not 2.3?

Reply:
I tried it but it didn't work. `0.2.2` has `storeAuthStateinCookie`, which was all I needed, so didn't try to dig deeper into why `0.2.3` didn't work.